### PR TITLE
Refs #1377 -- Added diamond and platinum members to the download page.

### PR DESCRIPTION
**Pull request by nessita**
Part of #1377.

### CODE REVIEW

**Comment by nessita** [comment]
Screenshot of proposed changes:
image

**Comment by sabderemane** [comment]
Hi @nessita, can you show how does it look with the dark mode ?

**Comment by nessita** [comment]
> Hi @nessita, can you show how does it look with the dark mode ?

Yes, sorry! Here it is (in all honesty, I did nothing special for the dark more... shall I?):

image

**Comment by sabderemane** [comment]
> Yes, sorry! Here it is (in all honesty, I did nothing special for the dark more... shall I?):

No worries, no seems to be ok, it's just to make sure it's not broken or not readable 🙂

**Comment by nessita** [comment]
I created a longer list of (fake) sponsors and noticed an issue that I have now sorted out. This is how it looks:
image

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -29,6 +29,7 @@ repos:
       rev: 23.3.0
       hooks:
           - id: black
+            args: ["--line-length", "88"]
```
Comment (sabderemane) [inline .pre-commit-config.yaml:32]: Why did you change this ?

File: djangoproject/scss/_style.scss [hunk 2]
```diff
@@ -3511,6 +3511,33 @@ ul.corporate-members li {
...
+	}
+
+	.small-cta {
+		width: 70%;
+		float: left;
+		vertical-align: top;
+		margin-left: 5%;
+		margin-right: 5%;
+		margin-bottom: 20px;
+	}
```
Comment (sabderemane) [inline djangoproject/scss/_style.scss:3536]: This might look small in mobile, do you have a screenshot to see if it's not the case?

**Review by sabderemane — COMMENTED** [review]
LGTM, just few questions 😄

File: .pre-commit-config.yaml (hunk 1 above)
Comment (nessita) [reply .pre-commit-config.yaml:32]: So, I was having resolution conflicts between black and isort. The former would wrap imports at 79 cols and the latter would expand them. So I was stuck in a loop looking like this, and the commit operation failed every time:

```
black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted releases/tests.py

All done! ✨ 🍰 ✨
1 file reformatted.

isort....................................................................Failed
- hook id: isort
- files were modified by this hook

Fixing /home/nessita/fellowship/djangoproject.com/releases/tests.py
```

The proposed solution reconciles isort and black regarding line length. If there is a better solution for this, I'm happy to change this! I could also prepare a separated PR if you prefer. What do you think?

File: djangoproject/scss/_style.scss (hunk 2 above)
Comment (nessita) [reply djangoproject/scss/_style.scss:3536]: Good point, Let me get that screenshot (I'll upload it ASAP). Though this class is the same as the `small-cta` used next to the "donation heart in the sidebar", so if we need to change this, we may want to change them both.

**Comment by nessita** [comment]
Attaching mobile screenshot:
image

File: .pre-commit-config.yaml (hunk 1 above)
Comment (sabderemane) [reply .pre-commit-config.yaml:32]: I understand but I think it's better to have it on a separated PR and ask review from djangoproject maintainers to make sure we all agree on this change 🙂

File: djangoproject/scss/_style.scss (hunk 2 above)
Comment (sabderemane) [reply djangoproject/scss/_style.scss:3536]: Hmm, I think it's fine for now, we'll see how to improve it later since there's already an ongoing project to redesign the site.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (nessita) [reply .pre-commit-config.yaml]: Makes sense. I have reverted and I was able to commit, so before I must've been mistaken or something else was going one :shrug: 

Thanks for your time!

**Review by sabderemane — APPROVED** [review]
